apps/codeforafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,91 @@
+import richText from "../fields/richText";
+
+const HeroTypes = [
+  {
+    name: "animatedHero",
+    label: "Animated Hero",
+  },
+  {
+    name: "staticHero",
+    label: "Static Hero",
+  },
+  {
+    name: "imageHero",
+    label: "Image Hero",
```

Comment:
Not sure about this. Hero is usually just one on the whole site & PageHeader is also very well understood. How are we handling this in charter.africa?

Reply:
@kilemensi I think we can make each its own block. This will also help solve the [above](https://github.com/CodeForAfrica/ui/pull/567#discussion_r1318480034)